agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -35,7 +35,9 @@ class backhaul_manager : public btl::transport_socket_thread {
     virtual bool init() override;
     virtual bool work() override;
 
-protected:
+    // For agent_ucc_listener
+    const std::string &get_controller_bridge_mac() { return controller_bridge_mac; }
```

Comment:
I think this commit is redundant. We are in control path, there is nothing wrong with doing a system call whenever we need the controller bridge MAC. If we do that every time we need it, we are also adding the infrastructure for being able to cope with dynamic changes to the main bridge, which is hardcoded everywhere currently.

Reply:
But there is no system call to get the remote controller bridge mac.
Only the backhaul manager holds it, and it gets it after receiving an auto-configuration search message response.
😕

Reply:
Then the name should change to `controller_almac`